shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Examples/EmptyBlock.hs

```diff
@@ -0,0 +1,80 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Shelley.Spec.Ledger.Examples.EmptyBlock
+  ( -- | = Empty Block Example
```

Comment:
Move this documentation to the module, rather than the example in the export list?